DocsClientYamlTestSuiteIT.test {yaml=reference/analysis/analyzers/lang-analyzer/line_1353} fails on 6.x with incorrect offset

Here's the relevant log:

```
  1> [2019-02-01T08:26:18,320][ERROR][o.e.s.DocsClientYamlTestSuiteIT] [test] This failing test was generated by documentation starting at reference/analysis/analyzers/lang-analyzer.asciidoc:line_1353. It may include many snippets. See docs/README.asciidoc for an explanation of test generation.
  2> REPRODUCE WITH: ./gradlew :docs:integTestRunner -Dtests.seed=D68E75796C96A039 -Dtests.class=org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT -Dtests.method="test {yaml=reference/analysis/analyzers/lang-analyzer/line_1353}" -Dtests.security.manager=true -Dtests.locale=nn-NO -Dtests.timezone=CST -Dtests.distribution=oss-zip -Dcompiler.java=11 -Druntime.java=8
FAILURE 2.60s | DocsClientYamlTestSuiteIT.test {yaml=reference/analysis/analyzers/lang-analyzer/line_1353} <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: Failure at [reference/analysis/analyzers/lang-analyzer:1316]: expected:<{token=⁉, start_offset=2245, end_offset=2246, type=<EMOJI>, position=6560}> but was:<{token=⁉, start_offset=2245, end_offset=2247, type=<EMOJI>, position=6560}>
   > 	at __randomizedtesting.SeedInfo.seed([D68E75796C96A039:5EDA4AA3C26ACDC1]:0)
   > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:406)
   > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:383)
   > 	at java.lang.Thread.run(Thread.java:748)
   > Caused by: java.lang.AssertionError: expected:<{token=⁉, start_offset=2245, end_offset=2246, type=<EMOJI>, position=6560}> but was:<{token=⁉, start_offset=2245, end_offset=2247, type=<EMOJI>, position=6560}>
   > 	at org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT$CompareAnalyzers.execute(DocsClientYamlTestSuiteIT.java:198)
   > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:399)
   > 	... 38 more
```

This *does* reproduce on 6.x for me with:

```
./gradlew :docs:integTestRunner -Dtests.seed=D68E75796C96A039 -Dtests.class=org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT -Dtests.method="test {yaml=reference/analysis/analyzers/lang-analyzer/line_1353}" -Dtests.security.manager=true -Dtests.locale=nn-NO -Dtests.timezone=CST -Dtests.distribution=oss-zip -Dcompiler.java=11 -Druntime.java=8
```

--- a/docs/reference/analysis/analyzers/lang-analyzer.asciidoc
+++ b/docs/reference/analysis/analyzers/lang-analyzer.asciidoc
@@ -1358,7 +1358,7 @@ PUT /persian_example
       "char_filter": {
         "zero_width_spaces": {
             "type":       "mapping",
-            "mappings": [ "\\u200C=> "] <1>
+            "mappings": [ "\\u200C=>\\u0020"] <1>
         }
       },
       "filter": {

